packages/fxa-auth-server/lib/subhub/client.js

```diff
@@ -207,6 +210,7 @@ module.exports = function(log, config) {
           if (err.statusCode === 400 || err.statusCode === 402) {
             throw error.rejectedSubscriptionPaymentToken(err.message, err);
           }
+          // TODO: update with whatever the error response is for missing display_name.
```

Comment:
I think you'll need to tweak the condition above as this is one possible meaning of the `400` response.

Ref:
https://github.com/mozilla/subhub/blob/e530fcb67339c8b41035ec2a67b06d23c4ef97b2/subhub/subhub_api.yaml#L133-L136

Reply:
(you can see elsewhere we resort to interrogation of `err.message`, I guess this will need more of the same sadly)

Reply:
Actually ignore me, given that it's `required` in the API here, I guess you don't need that condition after all?

Reply:
Ah, cool! Thanks, I'll remove the TODO